docs/java/index.md

```diff
@@ -12,20 +12,20 @@ sidebar: auto
 sidebarDepth: 0
 
 ---
-# Ethereum for Java Developers
+# Ethereum ♡ Java
```

Comment:
Let's keep this as "Ethereum for Java Developers", it's more direct & descriptive.

Reply:
Can't say I didn't try.  :)
I liked the heart emoji idea, but direct and descriptive is fine by me as well.